### PR TITLE
Make GenYaml common function

### PR DESCRIPTION
**Pull request by alexander-demichev**

### CODE REVIEW

**Comment by alexander-demichev** [comment]
Closes https://github.com/fusor/cpma/issues/167

File: pkg/transform/configmaps/configmaps_test.go [hunk 1]
```diff
@@ -1,9 +1,11 @@
-package configmaps
+package configmaps_test
```
Comment (jmrodri) [inline pkg/transform/configmaps/configmaps_test.go]: Why did you change the package? I find it easier to test when the tests are in the same package.

Comment (alexander-demichev) [reply pkg/transform/configmaps/configmaps_test.go]: If test belongs to `configmaps` I am getting import cycle error in this test.
```
import cycle not allowed in test
package github.com/fusor/cpma/pkg/transform/configmaps (test)
	imports github.com/fusor/cpma/pkg/transform
	imports github.com/fusor/cpma/pkg/transform/configmaps
```

Comment (alexander-demichev) [reply pkg/transform/configmaps/configmaps_test.go]: I thought about it, but I don't know what would be a good fit for `GenYAML`, maybe `utils/` ?

Comment (gildub) [reply pkg/transform/configmaps/configmaps_test.go]: It makes sense to test genYAML in transform since it's part of it!
